docs/getting-started.md

```diff
@@ -14,38 +14,46 @@ This guide describes how to get started with Flux with some examples demonstrati
  
 ## Setup
 
-The examples in this guide, along with examples found throughout this documentation, depend on a small application in the 
-`./examples/getting-started` directory in this repository. The examples assume that the application will be deployed to 
-a MarkLogic instance and that Flux will be run from the `./examples/getting-started/flux` directory as well. However, you are 
-free to install Flux anywhere and use the examples simply as a reference for running Flux on your own data. 
-
-### Obtaining Flux
-
-You can download the latest release of the Flux application zip from [the releases page](https://github.com/marklogic/flux/releases).
-
-If you would also like to use Flux to run the examples in this guide, please follow these steps:
-
-1. Clone this repository using git. 
-2. Extract the `flux-1.0-SNAPSHOT.zip` file into the `./examples/getting-started` directory in the cloned repository directory.
-This should result in an `./examples/getting-started/flux-1.0-SNAPSHOT` directory that contains the Flux software.
+You can download the latest release of the Flux application zip from [the latest Flux release page](https://github.com/marklogic/flux/releases). 
+The Flux application zip is titled `flux-1.0.0.zip`. You can extract this zip to any location on your 
+filesystem that you prefer.
 
 ### Deploying the example application
 
-The following steps will deploy a small application to your existing MarkLogic cluster, allowing you to run the 
-examples in this guide:
+The examples in this guide, along with examples found throughout this documentation, depend on a small MarkLogic 
+application that can be deployed to your own instance of MarkLogic server. The application can be downloaded from 
+[the latest Flux release page](https://github.com/marklogic/flux/releases) in a zip titled 
+`flux-getting-started-1.0-SNAPSHOT.zip`. To use Flux with this example application, perform the following steps:
 
-1. Clone this repository using git if you have not already.
-2. Run `cd examples/getting-started`.
-3. Create the file `gradle-local.properties` and add `mlPassword=your admin user password` to it.
-4. Change `mlHost` and `mlRestPort` as needed based on your MarkLogic installation.
+1. Extract the `flux-getting-started-1.0-SNAPSHOT.zip` file to any location on your local filesystem.
```

Comment:
The flux zip will be 1.0.0, but the example project zip will be a SNAPSHOT?
Is that intentional?

Reply:
That version number will get replaced, along with all the other 1.0-SNAPSHOT references.